src/java.ts

```diff
@@ -152,6 +155,7 @@ export async function handleJREProvisioning(
       url,
       method: 'GET',
       responseType: 'stream',
+      ...getHttpAgents(proxyUrl),
```

Comment:
we'll need this for all requests, wondering if we want to create an axios setup method for a few properties like the Authorization token and this proxy configuration so that we can reduce the needs for each request. WDYT

https://axios-http.com/docs/instance

Reply:
also baseUrl can be used

Reply:
I agree and I'm worried about doing it in this PR, as it requires https://github.com/SonarSource/sonar-scanner-npm/pull/116 to be merged or otherwise would require additional work after. Created https://sonarsource.atlassian.net/browse/SCANNPM-18 to implement it once both this and #116 are merged, if that sounds good to you too one of us can take it after